docs/custom_node_development.md

```diff
@@ -114,6 +114,8 @@ The best staring point for developing new custom nodes is via exploring or copyi
 
 The fully functional custom nodes are in:
 - [east-resnet50 OCR custom node](../src/custom_nodes/east_ocr)
+- [model zoo intel object detection custom node](../src/custom_nodes/model_zoo_intel_object_detection)
```

Comment:
I'm not sure about the naming in part: `model zoo intel`.
If it means that this node is meant to handle object detection from open models zoo models, then maybe we could go with: `open model zoo object detection custom node`

Reply:
There are 2 categories in model zoo: intel and public. We support most detection models from intel directory.